tests/phpunit/tests/blocks/updateIgnoredHookedBlocksPostMeta.php

```diff
@@ -142,12 +156,12 @@ public function test_update_ignored_hooked_blocks_postmeta_retains_content_if_no
 	/**
 	 * @ticket 60759
 	 */
-	public function test_update_ignored_hooked_blocks_postmeta_dont_modify_if_not_navigation() {
+	public function test_update_ignored_hooked_blocks_postmeta_dont_modify_if_incompatible_post_type() {
 		register_block_type(
 			'tests/my-block',
 			array(
 				'block_hooks' => array(
-					'core/navigation' => 'last_child',
+					'core/navigation-link' => 'after',
```

Comment:
If the anchor block is not the mocked root block (that represents the Nav or Template Part block), we don’t expect `ignoredHookedBlocks` to be set in the post meta, but in that anchor block’s block attributes (i.e. inline). So the test will pass — the post meta won’t be modified — but we can’t conclude that that happens because the post type is wrong 😬 

I noticed that our assertion has always been comparing the `post_content` (and we didn’t have one for the post meta). I think we should (also) assert that the post meta be unchanged, and change the anchor block type back to `core/navigation` and `last_child`.

Reply:
Done in aaea68d9a3.